### DISCUSSION TITLE
Imaging

### DISCUSSION THREAD

**Discussion by Ganarajr** [AxisCommunications/discussions#76] [VAPIX] [answered]
Is there a way to configure Imaging paramters like ExposureValue using heapi calls similar to
//myserver/axis-cgi/param.cgi?action=list&group=Properties.PTZ.PTZ

More specifically are any cgis specific to Imaging apis available.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @Ganarajr. Documentation regarding VAPIX can be found [here](https://www.axis.com/vapix-library/). Please see if you find your desired configuration there.
